func/omp/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ endfunction(omp_func)
 
 omp_func(checks checks.cpp false)
 omp_func(hellomp hellomp.cpp false)
+omp_func(create-parallel create-parallel.cpp false)
```

Comment:
I've chosen convention of underscores in cmake targets, .h and .cpp files so let's stick with that rather than hyphens. If you see any I've missed it's worth changing them to underscores.